<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>T3k Configurator(Factory Edition)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/T3kCfgFERes/resources/T3kCfgFE.ico</normaloff>:/T3kCfgFERes/resources/T3kCfgFE.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QStyleButton" name="btnSelectSensor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>12</y>
     <width>143</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Sensor...</string>
   </property>
  </widget>
  <widget class="QStyleButton" name="btnSideview">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>397</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>1. Sideview...</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStyleButton" name="btnDetection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>122</y>
     <width>397</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>2. Detection...</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStyleButton" name="btnBentAdjustment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>184</y>
     <width>397</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>3. Bent Adjustment...</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStyleButton" name="btnTouchSetting">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>246</y>
     <width>397</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>4. Touch Setting...</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStyleButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>328</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QStyleButton" name="btnTouchMark">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>398</y>
     <width>120</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Touch Mark
Window...</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtEdtDisplayFirmware">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>328</y>
     <width>151</width>
     <height>110</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	border-color: rgb(190, 190, 190);
	border-style: solid;
	border-width: 2px;
	background-color: rgb(240, 240, 240);
 }</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QStyleButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>328</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>309</y>
     <width>397</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblSoftlogicInfo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color : #403F95;
	font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>*Softlogic detected</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkSafeMode">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>284</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	color: rgb(80,80,80);
	font: bold 13px;
}

QCheckBox::indicator {
	width: 20px;
	height: 20px;
}

QCheckBox:hover {
	color: rgb(200, 80, 80);
}

QCheckBox:checked {
	color: rgb(240, 30, 30);
}</string>
   </property>
   <property name="text">
    <string>Safe Mode</string>
   </property>
  </widget>
  <widget class="QCmdAsyncManagerCtrl" name="cmdAsyncMngr" native="true">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>441</y>
     <width>300</width>
     <height>10</height>
    </rect>
   </property>
  </widget>
  <zorder>btnSideview</zorder>
  <zorder>btnSelectSensor</zorder>
  <zorder>btnDetection</zorder>
  <zorder>btnBentAdjustment</zorder>
  <zorder>btnTouchSetting</zorder>
  <zorder>btnReset</zorder>
  <zorder>btnTouchMark</zorder>
  <zorder>txtEdtDisplayFirmware</zorder>
  <zorder>btnExit</zorder>
  <zorder>line</zorder>
  <zorder>lblSoftlogicInfo</zorder>
  <zorder>chkSafeMode</zorder>
  <zorder>cmdAsyncMngr</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QStyleButton</class>
   <extends>QPushButton</extends>
   <header>QStyleButton.h</header>
  </customwidget>
  <customwidget>
   <class>QCmdAsyncManagerCtrl</class>
   <extends>QWidget</extends>
   <header>QCmdAsyncManagerCtrl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
